qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -198,7 +199,7 @@ def save_account(
 
         if not token or not isinstance(token, str):
             raise IBMQAccountCredentialsInvalidToken(
-                'Invalid token found: "%s" %s' % (token, type(token)))
+                'Invalid IBM Quantum Experience token found: "{}" {}.'.format(token, type(token)))
```

Comment:
nit - can you change the message to `Invalid IBM Quantum Experience token found: "{}" of type {}`?